src/manifest.model.yaml

```diff
@@ -40,11 +39,11 @@ models:
   download_url: https://github.com/deepimagej/models/releases/download/0.1/fru-net_sev_segmentation.zip
 
 - UNetDA:
-  config_url: https://raw.githubusercontent.com/subeeshvasu/hbp-DL-seg-codes/0.1.1/UNetDA.model.yaml
-  applications: [Ilastik, ImJoy]
-  download_url: https://github.com/subeeshvasu/hbp-DL-seg-codes/releases/download/0.1.1/UNetDA.model.zip
+  config_url: https://raw.githubusercontent.com/subeeshvasu/hbp-DL-seg-codes/0.1.2/UNetDA.model.yaml
```

Comment:
Note: I changed this to 0.1.2 because the generated json is at 0.1.2. There was a history update that we updated the json but not the yaml. We need to pay attention to this to make sure that we updated the yaml.